### ISSUE TITLE
Wiki/screenshots missing?

### ISSUE THREAD

**Issue by NullVoxPopuli** [phaazon/mind.nvim#43]
The readme has a couple links to a wiki and screenshots. Should those exist?

**Comment by phaazon** [comment]
Yep, I need to add them. Thanks for the reminder, I will do that on the weekend.

**Comment by NoahTheDuke** [comment]
I just installed mind.nvim and ran into the issue of "but how do I use it?" so I would love to see this expanded.

**Comment by mostafaqanbaryan** [comment]
I think that's not gonna happen

**Comment by GlitchyCrafting** [comment]
I would also appreciate a wiki or at least an example in the README.

**Comment by maggick** [comment]
Long weekend @phaazon? ;)
